plugins/locator/stdloc/descriptions/global_stdloc.xml

```diff
@@ -71,30 +71,35 @@
 						<group name="GridSearch">
 							<description>
 								Find the source location by evaluating the hypocenter probability
-								of each grid cell and returning the maximum likelihood hypocenter.
-								The source time is derived from the weighted average of arrival
+								of each grid point and returning the maximum likelihood hypocenter.
+								The source time is derived from the weighted average of the arrival
 								travel times.
 								The solution can be further improved combining it with the Least Squares
-								algorithm, which will then be run for each grid cell, using the cell
-								centroid as initial location estimate. In this case only few big cells
-								are required.
+								algorithm, which will then be run for each grid point, using it as
+								initial location estimate. In this case only few sparse points are
+								required.
 							</description>
-							<parameter name="center" type="list:string" default="auto,auto,5">
-								<description>Latitude,longitude,depth[km]. </description>
+							<parameter name="center" type="list:string" unit="deg,deg,km" default="auto,auto,20">
+								<description>Grid center defined as: latitude,longitude,depth [km]. The
```

Comment:
[km] can be dropped here, since unit tag is used.

Reply:
Good catch!